src/main/kotlin/com/fasterxml/jackson/module/kotlin/KotlinKeyDeserializers.kt

```diff
@@ -0,0 +1,72 @@
+package com.fasterxml.jackson.module.kotlin
+
+import com.fasterxml.jackson.core.JsonToken
+import com.fasterxml.jackson.core.exc.InputCoercionException
+import com.fasterxml.jackson.databind.*
+import com.fasterxml.jackson.databind.deser.std.StdKeyDeserializer
+import com.fasterxml.jackson.databind.deser.std.StdKeyDeserializers
+
+// The reason why key is treated as nullable is to match the tentative behavior of StdKeyDeserializer.
+// If StdKeyDeserializer is modified, need to modify this too.
+
+internal object UByteKeyDeserializer : StdKeyDeserializer(TYPE_SHORT, UByte::class.java) {
```

Comment:
In `KotlinDeserializers`, it was defined as `public`, but if `Deserializers` is `internal`, it seems preferable to make `Deserializer` `internal` as well, so I did.